nixos/doc/manual/administration/boot-problems.section.md

```diff
@@ -30,6 +30,12 @@ If NixOS fails to boot, there are a number of kernel command line parameters tha
 
 : Make systemd very verbose and send log messages to the console instead of the journal. For more parameters recognised by systemd, see systemd(1).
 
+In addition, these arguments are recognised by the live image only:
+
+`live.nixos.passwd=password`
+
+: Set the password for the `nixos` live user. This can be used for SSH access if there are issues using the terminal.
```

Comment:
```suggestion
`live.nixos.passwd=password` sets the password for the `nixos` live user. This can be used for SSH access if there are issues using the terminal.
```

Reply:
But that will prevent it from formatting as a list?
![image](https://user-images.githubusercontent.com/10761079/124347703-3ef2e980-dbe6-11eb-93e3-2e20e0c29622.png)
vs
![image](https://user-images.githubusercontent.com/10761079/124347712-487c5180-dbe6-11eb-82c6-46977a5b07d6.png)

Reply:
ok. your variant looks better.